java/test/jmri/jmrit/display/palette/ItemPaletteTest.java

```diff
@@ -14,15 +13,16 @@
  */
 public class ItemPaletteTest {
 
-    // allows creation in lamba expressions
+    // allows creation in lambda expressions
     private ItemPalette ip = null;
 
     @Test
     public void testShow() {
-        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        Assumptions.assumeFalse(GraphicsEnvironment.isHeadless());
```

Comment:
What is the benefit with `Assumptions` instead of `Assume`? I don't want you to change back, I ask since I want to learn.

Reply:
See my reply in general comment below (entered in IDE as a novelty)